### ISSUE TITLE
Update app packages reported by Snyk

### ISSUE THREAD

**Issue by mjtravers** [fecgov/fecfile-web-app#1632]
The Snyk vulnerability scanner reported a number of packages need upgrades.

For definitive list, look at the PRs listed in the app repo.

*   [x] zone.js
*   [ ] @types/ws
*   [x] primeng
*   [x] bootstrap
*   [x] @angular/animations
*   [x] @angular/common
*   [x] @angular/compiler
*   [x] @angular/core
*   [x] @angular/forms
*   [x] @angular/platform-browser
*   [x] @angular/platform-browsers-dynamic
*   [x] @angular/router
*   [x] gitpython

QA Acceptance Criteria

Screen shot of automated tests. Front end tests pass and QA smoke testing

### QA Notes ###
null

### DEV Notes ###
null

### Design ###
null

[FECFILE-517](https://fecgov.atlassian.net/browse/FECFILE-517)

**Comment by mjtravers** [comment]
This ticket has dependencies on the work being don in #1636. We're going to hit pause on this ticket until #1636 in complete and we can merge in that work. Placing ticket on hold and moving back to the backlog.
Current work is here when it is to be resumed: https://github.com/fecgov/fecfile-web-app/pull/1723

**Comment by mjtravers** [comment]
Placing ticket on hold. Will move forward to QA with #1636

**Comment by mjtravers** [comment]
Ticket no longer dependent on #1636 and can move forward on its own.
Passes CR, sending to QA.
QA Notes: No visual changes in the UI. Passes tests:
https://app.circleci.com/pipelines/github/fecgov/fecfile-web-app/5118/workflows/cc9ea31f-580f-4cb6-86ad-0ba5ae9935ad/jobs/17940
<img width="731" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM0NsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7070585d6f0bb609db3b68e8b3f9a8c93501a9d6/image.png" alt="image.png" />

**Comment by MitchellTCG** [comment]
PASSED QA
